packages/react/src/themes/teams-high-contrast/components/Alert/alertStyles.ts

```diff
@@ -18,16 +18,8 @@ const alertStyles: ComponentSlotStylesInput<AlertProps, AlertHighContrastVariabl
     ':hover': {
       backgroundColor: v.hoverBackgroundColor,
 
-      ':before': {
-        content: '""',
-        position: 'absolute',
-        top: 0,
-        right: 0,
-        bottom: 0,
-        left: 0,
-        border: `${pxToRem(2)} solid ${v.hoverBorderColor}`,
-        borderRadius: pxToRem(2),
-      },
+      // TODO: consider creating dedicated method for border styles on hover
```

Comment:
fyi @codepretty do we expect to have similar behavior for `:hover` border styles so that we can create something similar to `getBorderFocusStyles`?

Reply:
i believe so. there are a lot of hoverable elements that in hc will get a yellow background. some will need a border/outline too (eg icon button in attachment)

Reply:
@codepretty would be great to create a list of all of these in #1259 , already started